recipes/igl/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "igl" %}
+{% set version = "0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/geometryprocessing/libigl-python-bindings/archive/{{ version }}.tar.gz
+  sha256: cc749e5742986b20a440672a2a426b2b72598b179b411163768584009f44d445
+
+build:
+  number: 0
+  skip: True  # [win]
+  script:
+    - "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - git
+    - cmake
+
+  host:
+    - python
+    - pip
+    - setuptools
+    - libblas
+    - libcblas
+    - numpy >=1.9.1
```

Comment:
Use just `numpy` and let the global pinning take care of the minimum version.
With the `>=` here you are not building with `1.9`, you'll probably get `1.16` and then people with `<1.16` will have trouble using this package.

Reply:
ok